kuma/api/management/commands/publish.py

```diff
@@ -38,11 +38,25 @@ def add_arguments(self, parser):
             default=1000,
             help='Partition the work into tasks, with each task handling this '
                  'many documents (default=1000)')
+        parser.add_argument(
```

Comment:
I'm wondering how we might handle this for the `render_document` command (`kuma/wiki/management/commands/render_document.py`), where we don't have direct control of skipping the cache invalidation? I'm wondering if we might need a way to disable cache invalidation until commands like that have finished? I don't know. I need to think about that more. But it is a common case, for example, we periodically re-render all of the documents that use the `compat` macro (which is 10's of thousands of documents).